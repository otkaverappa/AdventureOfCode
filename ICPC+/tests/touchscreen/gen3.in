19
rkr 10
gix
jyw
eza
nnx
rnr
hgi
kjx
dqp
jwn
qpx
kvh 10
hgf
flg
fau
frh
ybp
glz
fqr
uao
jsd
hcb
qlh 10
wts
eau
yhm
igp
xma
wst
oov
eyp
jhu
kxh
sva 10
mad
hak
viq
kgf
mez
hun
cyn
uiu
ouu
vmp
ybr 10
dis
nda
gpi
lbp
mjj
zni
phs
jxm
ftb
xtc
owk 10
ilp
lre
veh
kqq
usg
ehp
ysn
kai
nxb
pnz
bwk 10
sje
yek
gpd
yjx
epe
uny
jxy
rnx
ucl
tgj
fys 10
lxy
wfn
bdy
ykn
ega
cpa
din
acr
lvx
wcy
qox 10
pkc
cnh
dlt
qpb
rtq
twa
gxc
zkz
xhc
xzq
uoc 10
ytp
hwc
bmr
cfk
vyh
vgg
agq
zfz
bcz
typ
vwi 10
mdg
oet
hja
ugb
dbh
jdp
bdu
bey
cyw
tvs
ciy 10
iye
dgp
gav
hdz
qoc
gpf
csm
bwm
xqh
ssr
scq 10
wiy
moa
jxd
int
lti
swb
ezx
qyp
ash
rkm
jjw 10
jxm
jgj
opw
hcq
quo
tbn
stn
itg
rnq
fyb
biy 10
prf
ygw
vna
nfw
dax
svr
fdm
lwz
ecy
ffg
gvx 10
ntf
lqv
ldc
hgd
hyy
ygd
krz
kxd
kdl
qli
pyd 10
xju
suz
wdi
zki
zio
evg
ehd
jri
ujt
eat
ixc 10
crw
boc
jpm
srx
gvs
obz
uls
ehd
amd
tud
wyu 10
uzk
wlc
lft
ino
acs
byd
vem
yea
byu
gwu
