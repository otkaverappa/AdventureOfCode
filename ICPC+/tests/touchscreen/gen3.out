rnr 3
gix 7
jyw 9
hgi 10
kjx 10
qpx 10
nnx 11
eza 13
jwn 15
dqp 16
hcb 5
frh 6
hgf 6
jsd 7
ybp 9
fau 10
uao 10
flg 11
fqr 12
glz 15
wst 10
wts 10
yhm 10
eyp 11
xma 11
eau 12
jhu 12
oov 12
igp 16
kxh 16
mez 10
viq 10
kgf 11
cyn 12
mad 12
hak 15
hun 15
vmp 16
uiu 19
ouu 20
ftb 8
nda 9
phs 9
jxm 10
lbp 10
mjj 10
xtc 11
dis 12
zni 12
gpi 13
kai 5
usg 6
ysn 7
lre 9
veh 10
kqq 11
nxb 11
ilp 12
ehp 14
pnz 15
yek 4
tgj 7
jxy 8
ucl 8
uny 13
gpd 14
rnx 16
sje 16
yjx 16
epe 18
ega 5
cpa 7
din 8
lvx 10
acr 11
bdy 11
wfn 11
ykn 11
wcy 13
lxy 16
qpb 4
zkz 5
xhc 8
dlt 9
rtq 10
pkc 12
twa 13
cnh 14
gxc 15
xzq 16
hwc 9
bmr 11
vyh 12
vgg 13
bcz 14
typ 14
ytp 14
agq 16
zfz 16
cfk 18
bdu 4
bey 4
jdp 8
mdg 9
dbh 10
cyw 11
oet 11
tvs 14
ugb 14
hja 17
gpf 8
dgp 9
xqh 9
csm 10
qoc 10
bwm 11
ssr 11
iye 12
gav 15
hdz 17
ezx 7
ash 9
jxd 9
swb 9
wiy 13
int 14
moa 15
qyp 16
rkm 17
lti 18
hcq 7
opw 7
rnq 7
jgj 8
itg 9
fyb 10
tbn 12
jxm 13
stn 14
quo 15
ffg 9
nfw 10
ecy 11
fdm 11
vna 11
ygw 11
svr 12
prf 14
dax 17
lwz 18
hgd 5
krz 6
ygd 6
kxd 7
ldc 7
ntf 8
hyy 11
lqv 11
kdl 13
qli 19
ehd 8
ujt 8
jri 12
evg 13
suz 13
eat 16
xju 17
wdi 18
zio 20
zki 20
obz 6
gvs 8
ehd 11
tud 11
uls 11
srx 12
amd 14
boc 14
crw 14
jpm 16
byu 5
gwu 8
vem 9
byd 10
ino 10
wlc 10
acs 13
lft 13
uzk 14
yea 14
